mod/developers/views/default/theme_sandbox/javascript/popup.php

```diff
@@ -2,33 +2,53 @@
 
 $ipsum = elgg_view('developers/ipsum');
 
-$link = elgg_view('output/url', array(
-	'text' => 'Popup content',
-	'href' => "#elgg-popup-test",
-	'rel' => 'popup',
-		));
+echo elgg_view_field([
+	'#type' => 'fieldset',
+	'align' => 'horizontal',
+	'fields' => [
+		[
+			'#type' => 'anchor',
+			'text' => 'Popup content',
+			'href' => "#elgg-popup-test",
+			'rel' => 'popup',
+		],
+		[
+			'#type' => 'button',
+			'class' => 'elgg-button elgg-button-submit',
+			'rel' => 'popup',
+			'data-href' => "#elgg-popup-test2",
+			'data-position' => json_encode(array(
+				'my' => 'left top',
+				'at' => 'left bottom',
+			)),
+			'value' => 'Load content in a popup',
+		],
+		[
+			'#type' => 'button',
+			'class' => 'elgg-button elgg-button-submit',
+			'rel' => 'popup',
+			'data-ajax-href' => 'ajax/view/theme_sandbox/components/tabs/ajax',
```

Comment:
Needs to be combined under single encoded `data-ajax` for both tabs and popup. Later we can add a helper AMD to do the dirty work.